creator-node/src/config.js

```diff
@@ -796,6 +802,18 @@ if (fs.existsSync(pathTo('contract-config.json'))) {
   })
 }
 
+// Set reconfigSPIdBlacklist based off of reconfigSPIdBlacklistString
+config.set(
+  'reconfigSPIdBlacklist',
+  config.get('reconfigSPIdBlacklistString') === ''
+    ? []
```

Comment:
should we just make this a set? setting this as an array and also making a set when we import this seems roundabout

Reply:
based off of the new logic change where we need to iter through these spIds to get the endpoint info, it's best to keep this as an array

Reply:
can iter through set as well but nbd either way